Engine/EditorUI/Panel/InspectorSubpanel/PanelMaterial.cpp

```diff
@@ -325,18 +316,10 @@ bool PanelMaterial::ShowMaterialTextureMap(std::shared_ptr<Material> material, M
 	if (ImGui::Button(ICON_FA_TIMES))
 	{
 		material->RemoveMaterialTexture(type);
-		if (type == Material::MaterialTextureType::NORMAL)
-		{
-			material->use_normal_map = false;
-		}
 		if (type == Material::MaterialTextureType::LIQUID)
 		{
 			material->use_liquid_map = false;
```

Comment:
You are not using it for liquid? There are only 2 variations at the moment?

Reply:
Liquid is not done for the moment

Reply:
This will be done in the new shader system